src/forms/CurrentExpenses.js

```diff
@@ -288,13 +288,14 @@ const ExpensesFormContent = function ({ client, current, time, setClientProperty
         : null
       }
 
-      {/** Wrapper here or else margins get added here, but not other
-      * places, making spacing hard to manage */}
-      <wrapper>
-        <FormHeading>Child Support</FormHeading>
-        <IntervalColumnHeadings type={type}/>
-        <CashFlowRow {...sharedProps} generic={'childSupportPaidOut'}> <strong>Legally obligated</strong> child support </CashFlowRow>
-      </wrapper>
+      { current.hasSnap
+        ? <wrapper>
+          <FormHeading>Child Support</FormHeading>
+          <IntervalColumnHeadings type={type}/>
+          <CashFlowRow {...sharedProps} generic={'childSupportPaidOut'}> <strong>Legally obligated</strong> child support </CashFlowRow>
+        </wrapper>
+        : null
+      }
```

Comment:
You can do `{hasSnap && <div />}`. `false` acts just like `null` here.

Reply:
Yeah, I've seen that around a bit. Wasn't sure exactly what it was doing or how. I have to say, even knowing that I find that syntax a bit harder to read, so I'm going to stay away from it for now, at least for this project.